src/bundle/ui-dev/src/modules/sub-items/components/table-view/table.view.component.js

```diff
@@ -181,16 +182,16 @@ export default class TableViewComponent extends Component {
                 return null;
             }
 
-            let className = TABLE_HEAD_CLASS;
             let onClick = null;
+            const className = createCssClassNames({
+                [TABLE_HEAD_CLASS]: true,
+                [`${TABLE_CELL_CLASS}--sortable`]: columnKey in SORTKEY_MAP,
+                [`${TABLE_CELL_CLASS}--sorted-asc`]: SORTKEY_MAP[columnKey] === sortClause && sortOrder === 'ascending',
+                [`${TABLE_CELL_CLASS}--sorted-desc`]: SORTKEY_MAP[columnKey] === sortClause && sortOrder === 'descending',
+                'c-table-view-item__cell--name': columnKey === 'name',
```

Comment:
I'm not sure if this is proper class name.
Does this element have a parent with class `c-table-view-item`?